run_all_tests.sh

```diff
@@ -1,12 +1,29 @@
 #!/usr/bin/env bash
-# Runs all tests.
+# Builds the project and runs all tests.
 #
 # A JVM will be selected by JAVA_HOME environment variable, or, if it is not set,
 # inferred from the java executable available on the path.
 
 # Fail immediately in case of errors and/or unset variables
 set -eu -o pipefail
 
-echo "Start running all EJB tests"
+# Import necessary environment variables (see the tests_profile header comment for details).
+source exonum-java-binding-parent/tests_profile
+
+echo "Start building the project with running all Java test"
+# Run unit and integration tests in ci-build profile. This profile includes:
+#  - Java unit & integration tests, including ci-only & slow non-critical tests,
+#    which are excluded in the default profile.
+#  - Checkstyle checks as errors.
+#  - Native unit & integration tests that do not require a JVM.
+#  - Test coverage information collection.
+# See build definitions of the modules for more.
+mvn install \
+  --activate-profiles ci-build \
+  -Drust.compiler.version="${RUST_COMPILER_VERSION}"
+
+echo "Start running EJB native tests"
 cd exonum-java-binding-parent
-./run_all_tests.sh
+# Run native integration tests that require prepared classpaths for fake classes.
```

Comment:
`run_all_tests.sh` in `ejb-parent/` remains as is to be able to build from that directory, is that correct?

Shall we put for now a NOTE saying to keep the two in sync?

Reply:
Yes, it is correct.
Ok, I'll write notes explaining this stuff.